import_images/handle_image.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# This script is intended to be run by packer, usage under any other
+# environment may behave badly.  It's purpose is to download a VM
+# image and a checksum file.  Verify the image's checksum matches.
+# If it does, convert the downloaded image into 'raw' format with
+# appropriate filename modification.
+#
+# The first argument is the file path and name for the downloaded image,
+# the second argument is the image download URL (ending in a filename).
+# The third argument is a the download URL for a checksum file containing
```

Comment:
Nit: pick one: `a the`.

Reply:
Whut? Is that TOO specific or something? :grinning: 

Thanks, I'll fix it.